app/components/post_draft/draft_input/index.tsx

```diff
@@ -19,6 +20,11 @@ type Props = {
     channelId: string;
     rootId?: string;
     currentUserId: string;
+    location: string;
+
+    // Post Props
+    postProps: Post['props'];
+    updatePostProps: (postProps: Post['props']) => void;
```

Comment:
I see no point of passing `postProps` being an entire object, this can easily be just a string called `priority` with values `undefined | 'none' | 'important' | 'urgent'`

Reply:
Agreed, I have started that way but going by the design, there seem to be more options in the next phase so I made changes to accommodate in the future.

<img width="376" alt="Screenshot 2022-10-28 at 21 46 21" src="https://user-images.githubusercontent.com/3680799/198729558-9057c0f3-64d6-4bb8-b4dc-6e40d055e974.png">